feat: Added onReady & onClose to the API (#28)
Typeform/embed: README.md

We use `position: fixed` to position our modal relative to its containing block established by the viewport. If one of the modal ancestors has a `transform`, `perspective`, or `filter` css property set to something other than `none` the positioning will be relative to it and probably not visible by the user.

## Tests

> In order to run visual tests, it is need an applitools key.

Reviewer: ```suggestion
In order to run visual tests, you need an applitools key.
```
